src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+
+import java.io.IOException;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+public class Storage {
+    private static DukeIO dukeIo = new DukeIO();
+    private ArrayList<Task> storedTasks;
+    private FileInputStream fis;
+    private ObjectInputStream ois;
+    private FileOutputStream fos;
+    private ObjectOutputStream o;
+
+    @SuppressWarnings("unchecked")
+    public ArrayList<Task> load() throws IOException, ClassNotFoundException {
+        fis = new FileInputStream("./data/duke.txt");
```

Comment:
Not sure if this would affect this but, what would happen if the file did not exist?